src/utilities/theme/utils.ts

```diff
@@ -1,5 +1,5 @@
 import tokens from '@shopify/polaris-tokens';
-import {colorFactory} from '@shopify/polaris-tokens/color-factory';
+import {colorFactory} from '@shopify/polaris-tokens/dist-modern/color-factory';
```

Comment:
the index in the dist-modern folder reexports the color factory. We should treat that index as the new entry point rather than diving into specific files

```suggestion
import {colorFactory} from '@shopify/polaris-tokens/dist-modern';
```